analyzers/src/SonarAnalyzer.CFG/LiveVariableAnalysis/RoslynLiveVariableAnalysis.cs

```diff
@@ -295,6 +279,14 @@ private void ProcessParameterReference(State state, IParameterReferenceOperation
         //    state.CapturedVariables.UnionWith(allCapturedSymbols);
         //}
 
+        private static ISymbol ParameterOrLocalSymbol(IOperation operation) =>
+            operation switch
+            {
+                var _ when IParameterReferenceOperationWrapper.IsInstance(operation) => IParameterReferenceOperationWrapper.FromOperation(operation).Parameter,
+                var _ when ILocalReferenceOperationWrapper.IsInstance(operation) => ILocalReferenceOperationWrapper.FromOperation(operation).Local,
+                _ => null
```

Comment:
It was probably added in a later commit, but `ProcessSimpleAssignment` needs to return `null` as a normal thing. For example, when dealing with `field = 42`, we need to return `null` because we don't want to track that.